pkg/limayaml/validate.go

```diff
@@ -372,6 +372,45 @@ func Validate(y *LimaYAML, warn bool) error {
 		}
 	}
 
+	if y.Rosetta.Enabled != nil {
+		if *y.Rosetta.Enabled && *y.VMType != VZ {
+			return fmt.Errorf("field `rosetta.enabled` can only be enabled for VMType %q; got %q", VZ, *y.VMType)
+		}
+	}
```

Comment:
Not sure why you use a nested condition, especially since you group the conditions differently in the next block. Why not just

```suggestion
	if y.Rosetta.Enabled != nil && *y.Rosetta.Enabled && *y.VMType != VZ {
		return fmt.Errorf("field `rosetta.enabled` can only be enabled for VMType %q; got %q", VZ, *y.VMType)
	}
```

Reply:
was when I was issues with the tests. I was trying to figure out what was wrong